stubs/greenlet/greenlet/_greenlet.pyi

```diff
@@ -0,0 +1,71 @@
+from collections.abc import Callable
+from contextvars import Context
+from types import FrameType, TracebackType
+from typing import Any, overload
+from typing_extensions import Literal, TypeAlias
+
+_TraceEvent: TypeAlias = Literal["switch", "throw"]
+_TraceCallback: TypeAlias = Callable[[_TraceEvent, tuple[greenlet, greenlet]], object]
+
+CLOCKS_PER_SEC: int
+GREENLET_USE_CONTEXT_VARS: bool
+GREENLET_USE_GC: bool
+GREENLET_USE_STANDARD_THREADING: bool
+GREENLET_USE_TRACING: bool
+# this is a PyCapsule, it may be used to pass the gevent C-API to another C-extension
+# there isn't a runtime type for this, since it's only an opaque wrapper around void*
+# but it's probably still better than pretending it doesn't exist, so people that need
+# to pass this around, can still pass it around without having to ignore type errors...
+_C_API: object
+
+class GreenletExit(BaseException): ...
+class error(Exception): ...
+
+class greenlet:
+    @property
+    def _stack_saved(self) -> int: ...
```

Comment:
Is it necessary to include this? We generally prefer to only stub the public API in typeshed, unless there's a good reason to make an exception

```suggestion
```

Reply:
I don't think it's necessary per se, but stubgen did include it, It doesn't seem particularly harmful to include it, since it's a read only attribute. But I don't really have a horse in this race, I didn't check if some of the libraries built on top of greenlet rely on this attribute.

Reply:
Stubgen can be a bit overzealous when it comes to adding attributes. Please remove it; it's easy to add things to the stub later if they're required, but very hard to remove them later if they're not required (it's a backwards-compatibility break for our users!). Some of our tests, meanwhile, assume that private names are fictitious things that don't exist at runtime, so we won't get any test failures if a private attribute is deleted at runtime, meaning it's much harder for us to keep private attributes up to date.